pkgs/top-level/all-packages.nix

```diff
@@ -23028,6 +23028,8 @@ in
 
   fityk = callPackage ../applications/science/misc/fityk { };
 
+  galario = callPackage ../applications/science/astronomy/galario { };
```

Comment:
This generally isn't done for python packages unless they have an executable binary. It's not particularly useful being able to request the installation of "some version of galario, built for some unspecified version of python" if you're going to want to import it as a python package from some other environment.

Reply:
In fact I'd suggest you move it to the `python-modules/` directory completely.

Reply:
I'm not sure about this. `galario` is a C++ library with an optional Python wrapper. Without this I don't think it would be possible to install the library without the Python wrapper.

Reply:
Then I'd move it to `pkgs/development/libraries`. If it's a library it doesn't belong in the applications directory.